### PR TITLE
tweak tab placement on the landing page

### PR DESCRIPTION
**Pull request by Docteh**
After all of this work tweaking the tabs, I got them to work a little better.

<details><summary>old stuff</summary>

image image image image

The CSS `text-overflow: nowrap; overflow: hidden;` does most of what I wanted.

I didn't make anything worse, tabs will draw over the scroll bar in 10.5.1 with a small window:
image
</details>

Have we considered moving the changelog and privacy policy to be part of the tabs on the right hand? It would be the interface less "busy"

### CODE REVIEW

**Comment by McGiverGim** [comment]
They are tabs, so it seems clean to me if they are next to the other, with few space between both,  like in the original.

**Comment by Docteh** [comment]
Yeah, I wasn't fully happy with it, by fiddling with transform-origin I can make the tabs have a variable width, but I hard code the left side of the tab. other options for transform-origin would move the item when I resized it. But maybe I can rotate the link, and then position the tab.

**Comment by McGiverGim** [comment]
Yes, I looked into it before and I ended reducing the Spanish translation that was bigger than the tab ;)

So if you find a good way to make it work always, perfect!!

**Comment by mikeller** [comment]
@Docteh: Moving them to the left hand tab list would be another way to address this - probably even better, as it is more consistent.

**Comment by Docteh** [comment]
I got additional tabs added, might need some work, should I just commit onto this branch?
image

File: src/js/tabs/static_tab.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+TABS.staticTab = {};
+TABS.staticTab.initialize = function (staticTabName, callback) {
+    var self = this;
+
+    if (GUI.active_tab != staticTabName) {
+        GUI.active_tab = staticTabName;
+    }
+    var tabFile = "./tabs/" + staticTabName + ".html";
+    if (staticTabName == 'changelog') { tabFile = './changelog.html'; }
```
Comment (mikeller) [inline src/js/tabs/static_tab.js]: We could probably avoid this hack if we changed where changelog is put by the gulp task.

**Comment by mikeller** [comment]
Good solution, I like the unified look.

We could / should probably look into spiffing up the look of the content of these two tabs a bit now, they look drab compared to the other tabs.

**Comment by Docteh** [comment]
gulp now puts `changelog.html` into tabs/

I guess instead of having the content take up the whole width, we should specify a maximum width, and then put something on the side, like a pretty background.

Changelog is fine to read, but lots of empty space. The privacy policy is probably annoying to read. 
Preview of 21:9
image

**Comment by Docteh** [comment]
@mikeller I think inspiring improvement is the best bet. I notice that osd-bg-1.jpg is a jpg complete with artifacts and therefore I intensely dislike it, and osd-bg-2.png is a blurry mess and therefore I intensely dislike it.
image

**Comment by mikeller** [comment]
I'll merge this as it solves the translation / tab size issue in a good way.

We can always go hunting for a better background image, and open a new pull request when we've got one.

**Comment by McGiverGim** [comment]
@Docteh @mikeller I never liked the top button, and I don't know if this is the expected behaviour, but when I press the "flash" button in the top (not the flash tab in the left) the result page has the two new tabs. Maybe it is interesting to have them (they have the changelog and privacy) but is strange to have only two tabs and not all of them.

**Comment by Docteh** [comment]
I'm really hoping nobody is keeping score on how many of my PRs directly result in reported issues... I didn't notice that. I'm not sure why clicking the firmware button would throw out tabs.

**Comment by McGiverGim** [comment]
We all make bugs in this nightmare mesh of code. Me, for example, I disabled one filter by error when saving in the PIDs tab and it wasn't detected until released ;-)

This was one of the reasons why I didn't like the flash button. Is a little different to the flash tab to do the same.

**Comment by mikeller** [comment]
I think the flash button in itself isn't the problem - the problem is that clicking it changes the tab list, and ends up in a different place than selecting the 'Firmware Flasher' tab - this should be fixed, and clicking the 'Update Firmware' button should be disconnecting if needed, and then select the 'Firmware Flasher' tab behind the scenes.

The 'Update Firmware' button was added to address the bad user experience of, when connected, first having to disconnect, then select the 'Firmware Flasher' tab, in order to be able to flash firmware. It also made firmware flashing more discoverable, stopping user questions on 'how can I flash firmware.

I think in a clean solution, we will only have two possible sets of tabs - 'not connected', including the firmware flasher, and 'connected', with some tabs potentially hidden if they are not supported / expert mode is off.

**Pull request merged**